### ISSUE TITLE
PaqSync does not update paq-nvim.

### ISSUE THREAD

**Issue by dshumakerUT** [savq/paq-nvim#102]
PaqSync updates all my paqs except paq-nvim.

**To Reproduce**
In a nvim session:

- Type `:PaqLogClean`
- Type `:PaqSync`
- Type `:PaqLogOpen`
- Type `:mes`

Output of PaqLogOpen is 
```
error: Your local changes to the following files would be overwritten by merge:
	lua/paq.lua
Please commit your changes or stash them before you merge.
Aborting
```
Output of :mes is 
```
Paq [1/25] (up-to-date) nvim-fzf
Paq [1/25] failed to update paq-nvim
Paq [2/25] (up-to-date) nvim-lspconfig
Paq [3/25] (up-to-date) fzf-lua
Paq [4/25] (up-to-date) nvim-autopairs
Paq [5/25] (up-to-date) nvim-tabline
Paq [6/25] (up-to-date) monokai.nvim
Paq [7/25] (up-to-date) nvim-dap
Paq [8/25] (up-to-date) vgit.nvim
Paq [9/25] (up-to-date) chadtree
Paq [10/25] (up-to-date) gitsigns.nvim
Paq [11/25] (up-to-date) vim-color-spring-night
Paq [12/25] (up-to-date) nvim-phpcsf
Paq [13/25] (up-to-date) nvim-web-devicons
Paq [14/25] (up-to-date) glow.nvim
Paq [15/25] (up-to-date) nvim-lsp-installer
Paq [16/25] (up-to-date) coq_nvim
Paq [17/25] (up-to-date) telescope-z.nvim
Paq [18/25] (up-to-date) indentLine
Paq [19/25] (up-to-date) telescope.nvim
Paq [20/25] (up-to-date) vista.vim
Paq [21/25] (up-to-date) vim-grepper
Paq [22/25] (up-to-date) nvim-treesitter
Paq [23/25] (up-to-date) plenary.nvim
Paq [24/25] (up-to-date) feline.nvim
```

**Expected behavior**
The expectation is that it also updates paq-nvim.

**Environment:**
 - `nvim --version`: v0.6.1
```
NVIM v0.6.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@Monterey

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.6.1/share/nvim"

```
- `git --version`: 2.34.1
- OS:   Mac OS 12.1

Nvim checkhealth:
```

nvim: health#nvim#check
========================================================================
## Configuration
  - OK: no issues found

## Performance
  - OK: Build type: Release

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=^H
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $TERM_PROGRAM='tmux'
  - INFO: $COLORTERM='truecolor'

## tmux
  - OK: escape-time: 1
  - INFO: Checking stuff
  - OK: focus-events: on
  - INFO: $TERM: screen-256color
  - WARNING: Neither Tc nor RGB capability set. True colors are disabled. |'termguicolors'| won't work properly.
    - ADVICE:
      - Put this in your ~/.tmux.conf and replace XXX by your $TERM outside of tmux:
          set-option -sa terminal-overrides ',XXX:RGB'
      - For older tmux versions use this instead:
          set-option -ga terminal-overrides ',XXX:Tc'

nvim-lsp-installer: require("nvim-lsp-installer.health").check()
========================================================================
## nvim-lsp-installer report
  - OK: neovim version >= 0.6.0
  - WARNING: **julia**: not available
  - OK: **bash**: `GNU bash, version 5.1.16(1)-release (x86_64-apple-darwin21.1.0)`
  - OK: **tar**: `bsdtar 3.5.1 - libarchive 3.5.1 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8 `
  - OK: **gzip**: `Apple gzip 352.40.6`
  - OK: **curl**: `curl 7.77.0 (x86_64-apple-darwin21.0) libcurl/7.77.0 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.42.0`
  - OK: **wget**: `GNU Wget 1.15 built on darwin14.0.0.`
  - OK: **python3**: `Python 3.9.10`
  - OK: **node**: `v17.3.1`
  - OK: **PHP**: `PHP 8.1.2 (cli) (built: Jan 21 2022 04:47:46) (NTS)`
  - OK: **Ruby**: `ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.x86_64-darwin21]`
  - OK: **sh**: `Ok`
  - OK: **Composer**: `Composer version 2.2.5 2022-01-21 17:25:52`
  - OK: **Go**: `go version go1.17.6 darwin/amd64`
  - OK: **RubyGem**: `3.0.3.1`
  - OK: **npm**: `8.3.0`
  - OK: **pip3**: `pip 21.3.1 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)`
  - OK: **java**: `Ok`
  - OK: **javac**: `Ok`

nvim-treesitter: require("nvim-treesitter.health").check()
========================================================================
## Installation
  - OK: `tree-sitter` found 0.20.4 (parser generator, only needed for :TSInstallFromGrammar)
  - OK: `node` found v17.3.1 (only needed for :TSInstallFromGrammar)
  - OK: `git` executable found.
  - OK: `cc` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl", "zig" }
    Version: Apple clang version 13.0.0 (clang-1300.0.29.30)
  - OK: Neovim was compiled with tree-sitter runtime ABI version 13 (required >=13). Parsers must be compatible with runtime ABI.

## Parser/Features H L F I J

  Legend: H[ighlight], L[ocals], F[olds], I[ndents], In[j]ections
         +) multiple parsers found, only one will be used
         x) errors found in the query, try to run :TSUpdate {lang}

provider: health#provider#check
========================================================================
## Clipboard (optional)
  - OK: Clipboard tool found: pbcopy

## Python 2 provider (optional)
  - INFO: `g:python_host_prog` is not set.  Searching for python2 in the environment.
  - INFO: Executable: /usr/bin/python2
  - INFO: Python version: 2.7.18
  - INFO: pynvim version: 0.4.0 (outdated; from ~/Library/Python/2.7/lib/python/site-packages/neovim)
  - WARNING: Latest pynvim is NOT installed: 0.4.3

## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Multiple python3 executables found.  Set `g:python3_host_prog` to avoid surprises.
  - INFO: Executable: /usr/local/bin/python3
  - INFO: Other python executable: /usr/bin/python3
  - INFO: Python version: 3.9.10
  - INFO: pynvim version: 0.4.3
  - OK: Latest pynvim is installed.

## Python virtualenv
  - OK: no $VIRTUAL_ENV

## Ruby provider (optional)
  - INFO: Ruby: ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.x86_64-darwin21]
  - WARNING: `neovim-ruby-host` not found.
    - ADVICE:
      - Run `gem install neovim` to ensure the neovim RubyGem is installed.
      - Run `gem environment` to ensure the gem bin directory is in $PATH.
      - If you are using rvm/rbenv/chruby, try "rehashing".
      - See :help |g:ruby_host_prog| for non-standard gem installations.

## Node.js provider (optional)
  - INFO: Node.js: v17.3.1
  - WARNING: Missing "neovim" npm (or yarn) package.
    - ADVICE:
      - Run in shell: npm install -g neovim
      - Run in shell (if you use yarn): yarn global add neovim

## Perl provider (optional)
  - INFO: perl executable: /usr/local/bin/perl
  - ERROR: Command error (job=36, exit code 2): `/usr/local/bin/perl -W '-MApp::cpanminus' -e ` (in '/Users/dan/work/unleashed/projects/afa/afa')
    stderr: Can't locate App/cpanminus.pm in @INC (you may need to install the App::cpanminus module) (@INC contains: /usr/local/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0/darwin-thread-multi-2level /usr/local/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0 /usr/local/Cellar/perl/5.34.0/lib/perl5/5.34.0/darwin-thread-multi-2level /usr/local/Cellar/perl/5.34.0/lib/perl5/5.34.0 /usr/local/lib/perl5/site_perl/5.34.0).BEGIN failed--compilation aborted.

telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - OK: fd: found fd 8.3.1

## ===== Installed extensions =====

## Telescope Extension: `z`
  - INFO: No healthcheck provided

vim.lsp: require("vim.lsp.health").check()
========================================================================
  - INFO: LSP log level : WARN
  - INFO: Log path: /Users/dan/.cache/nvim/lsp.log
  - INFO: Log size: 9 KB

vim.treesitter: require("vim.treesitter.health").check()
========================================================================
  - INFO: Runtime ABI version : 13
```

Contents of paq-nvim.lua
```lua
-----------------------------------------------------------
-- Plugin manager configuration file
-----------------------------------------------------------

-- Plugin manager: paq-nvim
-- https://github.com/savq/paq-nvim

vim.cmd 'packadd paq-nvim'            -- load paq
local paq = require('paq-nvim').paq   -- import module with `paq` function

-- Add packages
-- for package info see: init.lua (Lua modules)
require('paq'):setup({verbose=true}) {
  'savq/paq-nvim';  -- let paq manage itself
  'famiu/feline.nvim';  -- status line written in lua
  'kyazdani42/nvim-web-devicons';
  --  'kyazdani42/nvim-tcoq_nvimree.lua'; -- file explorer written lua
  'Yggdroot/indentLine'; --VIMSCRIPT This plugin is used for displaying thin vertical lines at each indentation level for code indented with spaces
  'liuchengxu/vista.vim'; -- a tags viewer, vista.vim can also be a symbol navigator similar
  'nvim-lua/plenary.nvim';  -- (required by telescope, gitsigns, vgit ) A Lua module for asynchronous programming using coroutines
  'nvim-treesitter/nvim-treesitter'; -- Treesitter configurations and abstraction layer for Neovim.
  'neovim/nvim-lspconfig'; -- A collection of common configurations for Neovim's built-in language server client. -- LUA replacement for Ale
--  'hrsh7th/nvim-cmp'; -- A completion engine plugin for neovim written in Lua
--  'hrsh7th/cmp-nvim-lsp'; -- nvim-cmp source for neovim builtin LSP client
--  'hrsh7th/cmp-path'; -- nvim-cmp source for filesystem paths.
--  'hrsh7th/cmp-buffer'; -- nvim-cmp source for buffer words.
--  'saadparwaiz1/cmp_luasnip'; -- luasnip completion source for nvim-cmp
--  'L3MON4D3/LuaSnip'; -- auto triggered snippets in LUA
  'windwp/nvim-autopairs'; -- A super powerful autopair for Neovim. It supports multiple characters.
  'tanvirtin/monokai.nvim'; -- color theme came from brainf+ck
  'tanvirtin/vgit.nvim'; -- visual git plugin to see diffs
  'lewis6991/gitsigns.nvim';  -- For git gutter signs
  'rhysd/vim-color-spring-night'; -- my old colortheme
  'ellisonleao/glow.nvim'; -- a glow markdown preview in vim
  -- {"nvim-telescope/telescope-fzy-native.nvim", run = "make"};
  'nvim-telescope/telescope.nvim'; -- is a highly extendable fuzzy finder over lists
  'nvim-telescope/telescope-z.nvim'; -- is an extension for telescope.nvim that provides its users with operating rupa/z or its compatibles
  'mhinz/vim-grepper'; --  Use your favorite grep tool (ag, ack, git grep, ripgrep, pt, sift, findstr, grep) to start an asynchronous search
  'vijaymarupudi/nvim-fzf'; -- An asynchronous Lua API for using fzf in Neovim
  'ibhagwan/fzf-lua'; -- This is my take on the original fzf.vim, written in lua for neovim 0.5
  'crispgm/nvim-tabline'; -- nvim port of tabline.vim, written in Lua
  'mfussenegger/nvim-dap'; -- Debugger plugin
  { 'ms-jpq/chadtree', branch=chad, run='python3 -m chadtree deps' };
  { 'ms-jpq/coq_nvim', branch=coq };
  'praem90/nvim-phpcsf'; -- phpcs and phpcbf executables
  'williamboman/nvim-lsp-installer'; -- Could help with language servers.
}

```

**Comment by yochem** [comment]
Try:
```
$ cd ${XDG_DATA_HOME:-~/.local/share}/nvim/site/pack/paqs/start/paq-nvim && git checkout -- .
```
And then run PaqSync again.

Assuming you are on a unix system with bash/zsh.

**Comment by saccarosium** [comment]
I pretty sure this is solved after 2.0. Feel free to reopen if you still have the issue.
